code/modules/reagents/Chemistry-Reagents/acid.dm

```diff
@@ -6,8 +6,8 @@
 	color = "#db5008"
 	metabolism = REM * 2
 	touch_met = 50 // It's acid!
-	var/power = 2.5
-	var/meltdose = 10 // How much is needed to melt
+	var/power = 1
```

Comment:
these are very large changes, why so?  try to do smaller increments

Reply:
I've tested these changes extensively. Even something as 1.5 will still put someone in immediate soft-crit.

60 units is still enough to put someone into permanent hardcrit and passed out state from pain/shock.

Reply:
I just now noticed that you meant the other acids in specific.

So, Triflic Acid is still the biggest one available, but it's only natural that lesser Acids, such as sulphuric or whatever have less power.

The original was way too strong.